### PR TITLE
Certora/accounts update

### PR DESCRIPTION
**Pull request by shellygr**
### Description

- An update of the Certora spec for the updated Accounts contract
- A refresh of the Governance rules
- Extra rules for both Accounts and Governance
- CI sync and time improvements

Before changing from draft to actual PR:
- [x] cleanup apply harness
- [x] align locked gold spec to run from script too
- [x] go over arguments in scripts
- [x] README for governance specs

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -817,16 +818,44 @@ jobs:
             wget https://github.com/ethereum/solidity/releases/download/v0.5.13/solc-static-linux
             chmod +x solc-static-linux
             sudo mv solc-static-linux /usr/bin/solc
+      - run: echo ${CIRCLE_NODE_INDEX}
```
Comment (yorhodes) [inline .circleci/config.yml:821]: do we need this still?

File: packages/protocol/specs/accountsPrivileged.spec [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  || f.selector == setWalletAddress(address,uint8,bytes32,bytes32).selector
+  || f.selector == createAccount().selector
+  || f.selector == completeSignerAuthorization(address,bytes32).selector
+  || f.selector == removeValidatorSigner().selector
+  || f.selector == authorizeValidatorSigner(address,uint8,bytes32,bytes32).selector
+  || f.selector == setName(string).selector
+  || f.selector == initialize(address).selector
+  || f.selector == removeDefaultSigner(bytes32).selector
+  || f.selector == removeSigner(address,bytes32).selector
+  || f.selector == setEip712DomainSeparator().selector
```
Comment (yorhodes) [inline packages/protocol/specs/accountsPrivileged.spec:24]: ✅

File: packages/protocol/specs/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+### LockedGold
+- `lockedGold.spec` - general rules for the LockedGold contract.
+- `locked_gold_linked.spec` - rules for LockedGold that are coupled with Accounts.
+
+### Governance
+- `governance.spec` - general rules for the Governance contract. (Not exhaustive.)
+- `governancePrivileged.spec` - rule for identifying the privileged subset of functions, that is, functions that can only be executed by a single party.
+- `governance_with_dequeue.spec` - rules for Governance that require reasoning about the dequeue mechanism
+- `governance_old_rules.spec` - rules used in the 2019 formal verification project, and require some re-formulation.
```
Comment (yorhodes) [inline packages/protocol/specs/README.md:19]: this is great, thank you!

**Review by yorhodes — APPROVED** [review]

**Review by asaj — APPROVED** [review]
Approving config.yml changes

**Pull request merged**